cmd/auth-handler.go

```diff
@@ -125,7 +125,7 @@ func checkRequestAuthType(r *http.Request, bucket, policyAction, region string)
 
```

Comment:
An entry for each of StringLike, StringNotLike and AWS:Referer is already present as part of this PR. Let me know if the document needs additional changes.